### PR TITLE
Update branch manager whitelist

### PR DESCRIPTION
**Pull request by cblecker**
- Remove @k8s-merge-robot (RIP Submit Queue 🎉 )
- Transition 1.12 over to @feiskyer 
- Add @dougm, @hoegaarden, and @idealhack to 1.13 branch managers

cc: @AishSundar 
/assign @amwat @BenTheElder

### CODE REVIEW

File: prow/plugins.yaml [hunk 1]
```diff
@@ -174,9 +173,11 @@ slack:
...
             - foxish # 1.11 patch release manager
         release-1.12:
-            - dougm # 1.12 branch manager
-            - hoegaarden # 1.12 branch manager shadow
-            - idealhack # 1.12 branch manager shadow
+            - feiskyer # 1.12 patch release manager
+        release-1.13:
+            - dougm # 1.13 branch manager
+            - hoegaarden # 1.13 branch manager shadow
+            - idealhack # 1.13 branch manager shadow
```
[the remark this answers is not in the document]
Comment (cblecker) [reply prow/plugins.yaml]: We usually swap them over to the patch manager *after* the .0 release.

You’re wanting to add them right away? This only impacts the warning to k-dev, not the actual permissions.

[the remark this answers is not in the document]
Comment (AishSundar) [reply prow/plugins.yaml]: Ah I see. I wasn't aware of the set process here. Sounds good then, will remove the hold

**Comment by idealhack** [comment]
thanks, just know the existence of this whitelist :)

**Pull request merged**